### PR TITLE
added seed

### LINKED ISSUE #216: Jaccard similarity doesn't handle custom seeds
If two sketches with custom seeds are loaded and passed to `theta_jaccard_similarity::jaccard()`, an exception is generated for `seed hash mismatch`. It looks like the problem is that Jaccard builds unions and intersections but doesn't have a way to pass the relevant see to those sketches.

### PR DESCRIPTION
**Pull request by AlexanderSaydakov**
fixes #216

### CODE REVIEW

File: theta/test/theta_jaccard_similarity_test.cpp [hunk 1]
```diff
@@ -140,4 +179,21 @@ TEST_CASE("theta jaccard: dissimilarity test", "[theta_sketch]") {
...
+  const uint64_t seed = 1234;
+
+  auto expected = update_theta_sketch::builder().set_lg_k(min_lg_k).set_seed(seed).build();
+  for (int i = 0; i < u1; ++i) expected.update(i);
+
+  auto actual = update_theta_sketch::builder().set_lg_k(min_lg_k).set_seed(seed).build();
+  for (int i = 0; i < u2; ++i) actual.update(i);
+
+  REQUIRE(theta_jaccard_similarity::dissimilarity_test(actual, expected, threshold, seed));
+  REQUIRE_FALSE(theta_jaccard_similarity::dissimilarity_test(actual, actual, threshold));
```
Comment (jmalkin) [inline theta/test/theta_jaccard_similarity_test.cpp:196]: should this line have the seed passed in, too?

Comment (jmalkin) [reply theta/test/theta_jaccard_similarity_test.cpp:196]: oh, nevermind, misread

**Comment by jmalkin** [comment]
👍

**Review by jmalkin — APPROVED** [review]

File: theta/test/theta_jaccard_similarity_test.cpp (hunk 1 above)
Comment (AlexanderSaydakov) [reply theta/test/theta_jaccard_similarity_test.cpp:196]: yes. the test did not fail because the implementation checks for identical input upfront

**Pull request merged**